src/main/java/frc/robot/RobotContainer.java

```diff
@@ -225,6 +229,26 @@ private void configureBindings() {
                         })));
   }
 
+  private boolean IsEndGame() { // Endgame starts when match time is 20 seconds or less
+    return DriverStation.getMatchTime() <= 20 && DriverStation.isAutonomous() == Boolean.FALSE;
+  }
+
+  private void configureLEDTriggers() {
+    // Set "defalt" color for alliance to red or blue
+    new Trigger(() -> DriverStation.isFMSAttached())
+        .onTrue(Commands.runOnce(() -> LED.getalliance()));
+    // If intake Has Game Piece set color to orange
+    new Trigger(() -> intake.getHasGamePiece()).onTrue(Commands.runOnce(() -> LED.LEDDO("o")));
```

Comment:
One part that makes the code not "read like a sentence" is the "o" character to mean orange or "game piece". We can talk today about how to restructure things so it reads more like:

`LED.LEDDO(LedConstants.GAME_PIECE)`

Plus we might not want it to be orange this year! :)

Reply:
I think the implementation would need to look like the constants files. So something like LedConstants.java. Here's an example of another constants file: https://github.com/AEMBOT/FRC_2025/blob/main/src/main/java/frc/robot/constants/DriveConstants.java